js/example-frontend-js/build.gradle.kts

```diff
@@ -21,7 +21,13 @@ kotlin {
     }
 }
 
+// For kotlinx-html
+repositories {
+    maven("https://kotlin.bintray.com/kotlinx")
```

Comment:
It has moved to space `maven("https://maven.pkg.jetbrains.space/public/p/kotlinx-html/maven")`